examples/kitchen-sink/pages/page/[filename].tsx

```diff
@@ -1,7 +1,7 @@
 import { InferGetStaticPropsType } from 'next'
 import { Json } from '../../components/json'
 import { useTina } from 'tinacms/dist/react'
-import client from '../../.tina/__generated__/client'
+import { dbConnection } from '../../lib/db'
```

Comment:
We don't have parallel changes like this for basic-iframe. I attempted to, but ran into issues with the shape being returned. 

Would it make sense to pull the changes in basic-iframe into kitchen-sink (and revert basic-iframe)? The only reason I put them in basic-iframe and not kitchen-sink was because I wanted to provide an example while avoiding stomping on Jeff's classiclevel changes, which we are now discarding.